Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2_test.py

```diff
@@ -0,0 +1,171 @@
+    client = Client(
+        url=MOCK_URL,
+        use_ssl=False,
+        use_proxy=False,
+        fetch_interval_hours=1)
+    status = reload_command(client)
+    assert status.readable_output == "updated successfully"
+
+
+STANDARD_NOT_LOADED_MSG = 'OpenPhish Database Status\nDatabase not loaded.\n'
+STANDARD_4_LOADED_MSG = "OpenPhish Database Status\n" \
+                        "Total **4** URLs loaded.\n" \
+                        "Last load time **Thu Oct 01 2020 06:00:00 (UTC)**\n"
+CONTEXT_MOCK_WITH_STATUS = [
+    ({}, STANDARD_NOT_LOADED_MSG),  # case no data in memory
+    ({"list": [], "timestamp": "1601532000000"},
+     STANDARD_NOT_LOADED_MSG),  # case no timestamp and list is emtpy
+    (
+        {"list": ['http://www.niccakorea.com/board/index.html',
+                  'http://lloyds.settlemypayee.uk',
+                  'https://whatsapp-chat02.zzux.com',
+                  'http://dd0ddddddcuser.ey.r.appspot.com'],
+         "timestamp": "1601532000000"},  # datetime(2020, 10, 1, 10, 00, 00, 0) - timedelta(hours=1)}
+        STANDARD_4_LOADED_MSG)
+]
+
+
+@pytest.mark.parametrize('data,expected_result', CONTEXT_MOCK_WITH_STATUS)
+@freeze_time("1993-06-17 11:00:00 GMT")
+def test_status_command(mocker, data, expected_result):
+    """
+    Given:
+        - Integration context
+    When:
+        - After status command
+    Then:
+        - Returns number of loaded urls if data was loaded.
+        - Otherwise, returns Database not loaded.
+    """
+    client = Client(MOCK_URL, True, False, 1)
+    mocker.patch.object(demisto, "getIntegrationContext", return_value=data)
+    status = status_command(client)
+    assert status.readable_output == expected_result
+
+
+CONTEXT_MOCK_WITH_URL = [
+    ({'url': 'http://lloyds.settlemypayee.uk'},
+     {"list": ['http://www.niccakorea.com/board/index.html',
+               'http://lloyds.settlemypayee.uk',
+               'https://whatsapp-chat02.zzux.com',
+               'http://dd0ddddddcuser.ey.r.appspot.com'],
+      "timestamp": "1601532000000"},
+     {'URL': [{'Data': 'http://lloyds.settlemypayee.uk',
+               'Malicious': {'Vendor': 'OpenPhish', 'Description': 'Match found in OpenPhish database'}}],
+      'DBOTSCORE': [{'Indicator': 'http://lloyds.settlemypayee.uk', 'Type': 'url', 'Vendor': 'OpenPhish', 'Score': 3}]}
+     ),
+    ({'url': 'http://goo.co'},
+     {"list": ['http://www.niccakorea.com/board/index.html',
+               'http://lloyds.settlemypayee.uk',
+               'https://whatsapp-chat02.zzux.com',
+               'http://dd0ddddddcuser.ey.r.appspot.com'],
+      "timestamp": "1601532000000"},
+     {'URL': [{'Data': 'http://goo.co'}],
+      'DBOTSCORE': [{'Indicator': 'http://goo.co', 'Type': 'url', 'Vendor': 'OpenPhish', 'Score': 0}]}
+     )
+]
+
+
+@pytest.mark.parametrize('url,context,expected_results', CONTEXT_MOCK_WITH_URL)
+def test_url_command(mocker, url, context, expected_results):
```

Comment:
please add given: a url, when: "mocking the integration context data, running url_command", then: .... validating...